### DISCUSSION TITLE
change operators

### DISCUSSION THREAD

**Discussion by kellywang1234** [react-querybuilder/react-querybuilder#628] [Q&A]
https://stackblitz.com/edit/github-q8ems4-dtfvgf?file=src%2FApp.tsx,package.json
this is sandbox example, could you please help me filter the operators to only = and != ? Thank you so much!

**Comment by jakeboone02** [comment]
In the context of your implementation, the easiest way is to use the `getOperators` prop ([documentation for v5 here](https://react-querybuilder.js.org/docs/5/api/querybuilder#getoperators)).

[Forked sandbox here](https://stackblitz.com/edit/github-q8ems4-xkgwty?file=src%2FApp.tsx).

This is the function I used in the forked sandbox:

```
const getOperators = (field: string) =>
  field === 'max|age'
    ? defaultOperators.filter(op => op.name === '=' || op.name === '!=')
    : defaultOperators;
```

Also, not that it matters much for what you're doing, but your RQB version is out of date. Version `6.5.4` is the latest official release, and version 7 is coming soon (`7.0.0-alpha.5` seems to be fairly stable).
